### ISSUE TITLE
Ballistic vest has unnaturally high protection.

### ISSUE THREAD

**Issue by Ezsk** [CleverRaven/Cataclysm-DDA#36682]
Ballistic vest has unnaturally high protection.

# Detail
I'm not sure this is correct implementation or not. However, This value is better than basic power armor. I feel this is unnatural.
Ballistic vest has "material_thickness: 20". This is highest value of the game and it provide a cut protection 90.

Name | cut protection | encumbrance(fit) | weight
---------------------- |--------------|------------|--------
leather jacket | 6 | 15(7) | 1450 g
armored leather jacket | 9 | 24(14) | 2180 g
leather vest | 6 | 8(4) | 997 g
armored leather vest | 9 | 24(14) | 1210 g
scrap cuirass | 10 | 18 | 3366 g 
cuirass | 16 | 20(10) | 4200 g
ballistic vest | 90 | 10 | 9911 g

# Screenshots
![the stat](https://i.imgur.com/Iuk5uKP.png)

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.D-10974-g828501c
 - Graphics version: Tiles
 - Mods loaded: none

**Comment by ymber** [comment]
ESAPI plates are 20-30mm thick and they're NIJ IV rated. They should be able to stop a .30-06 M2 round or several 7.62x51mm rounds. A lot of the armors in the game have armor values much lower than they should.

**Comment by GGgatherer** [comment]
AFAIU the question is, is it that BV is too good or power armor - along with many regular armor pieces - is too bad?

**Comment by tenmillimaster** [comment]
> AFAIU the question is, is it that BV is too good or power armor - along with many regular armor pieces - is too bad?

Power armor isn't necessarily the end-all be-all armor. It's just the combination of an exosuit and additional body armor, not an impervious suit for wading into death.

Dedicated ballistic armor should outclass power armor.

**Comment by GGgatherer** [comment]
> Power armor isn't necessarily the end-all be-all armor. It's just the combination of an exosuit and additional body armor, not an impervious suit for wading into death.

I disagree.
What you are talking about is RM-13 combat armor. Which probably could be outclassed by some regular dedicated ballistic armors, but even that is questionable.
But PA is, well, PA. Surely, not "an impervious suit for wading into death", but it is the thing which is made for granting as much protection from weapons as possible. If it didn't grant better protection than regular vests, what would even be the point of it? It already slows you down significantly.

**Comment by Ezsk** [comment]
Even the cut protection value is suitable, 10 kg of body armor is not considered as easy to wear as leather jacket. How do you think about increase the encumbrance ?
for example,  MBR vests (steel plating) has 30 encumbrance.

**Comment by tenmillimaster** [comment]
What you are talking about is RM-13 combat armor

Experimental combat armors that are hanging onto the the game by a thread are probably not a good metric to judge other armors against.

**Comment by Solusphere** [comment]
RM-13 combat armor is "powered armor" not power armor. It's non-rigid and doesn't have a powered exoskeleton in the conventional sense. Per it's startup procedure it's making use of some  "Electro-reactive armor system" to achieve it's protection levels when powered. When unpowered, it's bash and cut protection are only 4 and 10 respectively, and even when it is powered it's protection scores are the same as an MBR vest with ceramic inserts, albeit with better coverage and lower encumbrance.

**Comment by Amoebka** [comment]
Doesn't really matter how much protection it has, 85% coverage makes it not worth using.
And yes, it's the OTHER armours that are wrong. A ballistic vest should be able to sometimes stop bullets, as crazy at is sounds.

**Comment by RobertoVGoulart** [comment]
A ballistic vest should be able to stop bullets easily to begin with, but lose protection value very fast as it takes damage, since its protection value comes from the materials used in it.
A power armor should have as much protection, or maybe even a little less, when powered, but since that protection comes from a "reactive" mechanism, it should take less structural damage and lose less protection as it does.

**Comment by Ker-Nes** [comment]
>  A power armor should have as much protection, or maybe even a little less, when powered, but since that protection comes from a "reactive" mechanism, it should take less structural damage and lose less protection as it does.

You are thinking powered armor, like the already mentioned RM-13. The major thing about the power armor is that, it being powered and thus the person not needing to carry the weight (and depending on how good is the tech, even get nearly as good movement range and reflexes as if he was naked), thus it can "afford" to have boatloads of armor that an unaided person simply could not wear around them and stay mobile.

**Comment by Amoebka** [comment]
Well except power armor we have in game doesn't lose weight when turned on, oly encumbrance.

**Comment by Ezsk** [comment]
Aside from power armor, I think this thread should focus on ballistic vests.

To summarize the opinion, cut protection is seems to appropriate. because ballistic vests were made to be bulletproof.
However, Would it be possible to adjust the game balance while respecting the actual ballistic vest? I think that is necessary thinking. 

And then, How is the encumbrance? What is the different from MBR vest?

**Comment by ymber** [comment]
MBR vests are some fictional thing from old cata. It's hard to tell what they're supposed to represent and they shouldn't be used as a reference point. The coverage of the ballistic vest should possibly be 90%-95% given that areas not covered by plates are still well armored but I went with 85% because our coverage model doesn't handle different armor levels. An encumbrance value of 10 is appropriate judging from the impact of different encumbrance levels.

**Comment by Ezsk** [comment]
The wearing comfort is not described in the actual catalog specifications. I think an encumbrance can make use of balancing game.

Typically, Ceramic plates hinder the bending of body joints, and it shift the balance of weight. In fact, some Marines complained that they were "too heavy and impractical and lack mobility". These disadvantage should be reflect as exchange of protection advantage.

here is the some samples

Name | cut protection | encumbrance(fit) | weight
---------------------- |--------------|------------|--------
leather jacket | 6 | 15(7) | 1450 g
armored leather jacket | 9 | 24(14) | 2180 g
leather vest | 6 | 8(4) | 997 g
armored leather vest | 9 | 24(14) | 1210 g
scrap cuirass | 10 | 18 | 3366 g 
cuirass | 16 | 20(10) | 4200 g
ballistic vest | 90 | 10 | 9911 g

IMO, SPCS without ESAPI would be 10 encumbrance. but SPCS  with full of level 4 ESAPI would be 20-30 encumbrance.

Also, the many fictitious monster's attack power has been adjusted based on classic old cata armor. I am concerned that new armor based on realism will break the game balance. In any case, We need a new standard for value of attack and defense.

**Comment by Ezsk** [comment]
> MBR vests are some fictional thing from old cata. It's hard to tell what they're supposed to represent and they shouldn't be used as a reference point.

The Ballistic vest you implemented is known as kind of MBR vest. I think we should adjust existing MBR vest instead of adding new ones.
Performance tuning of attack & defence requires consistency anyway.

**Comment by ymber** [comment]
You're quoting comments on the IOTV. The SPCS was specifically required to be much less encumbering than the IOTV because of the operational needs statement from Afghanistan. Looking more at the requirements and performance in SPD-VII I think it's exactly right where it is with 10 encumbrance and 85% coverage. I intend to remove all the current MBR vests and replace them with MOLLE pouches and items that reflect real armor.

**Comment by Ker-Nes** [comment]
> You're quoting comments on the IOTV. The SPCS was specifically required to be much less encumbering than the IOTV because of the operational needs statement from Afghanistan. Looking more at the requirements and performance in SPD-VII I think it's exactly right where it is with 10 encumbrance and 85% coverage. I intend to remove all the current MBR vests and replace them with MOLLE pouches and items that reflect real armor.

Why? The current MBR vests are perfectly well suited to represent a ballistic vest that has pouches for further slots for armored plates, and even if you want to make so it gets separated into the armored vest and the plate carriers separately, there's little reason not to use the already existing item, just with a few changes to protection values. 

In a similar note, no, 90 cut protection is simply ridiculous even so. That's not merely bullet resistant to a type III or even a type IV standard, but, given the damages in the game, capable of turning a _point blank_ shot with a .50 BMG into something that just kinda hurts.

**Comment by Ker-Nes** [comment]
> Well except power armor we have in game doesn't lose weight when turned on, only encumbrance.

It probably should be heavier, but increase carrying capacity (or even directly strength, if assuming the armor is capable of fast enough movement to also impart greater strength when using melee attacks and to other strength based activities) when worn and turned on.

**Comment by Ezsk** [comment]
> I think it's exactly right where it is with 10 encumbrance and 85% coverage. 

I thought and you thought then what is right. We should objective discussion.

The number of encumbrance does not exist in real. So, we should judge by relative from the standard. I choose a leather vest for the standard as experimental. because we can easily to imagine how is wearing feel.

and, here is a 1kg leather vest has 4-8 encumbrance. and 1.2kg metal paded vest is 14-24. this difference (10-16) is the measure tape of the encumbrance in cata.
(oh I think padded vest should reduce the encumbrance a bit. We should find a better measure tape but aside it)

and then, how is 9.9kg vest is ?
Of Course there are many differences. leather and nylon, modern scientific design. but physically weight will not cancel it. maybe it is not 10.
If you don't think so, Please explain me relatively and logically.

**Comment by crazykiddeath** [comment]
>  Changing bullet damage and how they work is probably a much better option. Maybe a damage multiplier for what manages to go through armor (perhaps with a top limit of what that damage can be, to represent over penetration?)? Or a new type of damage, to represent things with little weight moving at very high speeds, in contrast to things with higher weight moving at lower speeds?

I'd say this might be the best way to go about it, it would also allow a rebalance of other existing armours to better reflect their true effectiveness against other types of damage.

**Comment by ymber** [comment]
I'm not concerned with what values other items use. We can and should derive encumbrance values from real information about the item. The SPCS is relatively heavy at 10kg but it has quite good weight distribution. It's pretty well designed for wearer comfort and the descriptions of its performance in SPD-VII indicate that the impact of 10 encumbrance closely reflects the impact of wearing the SPCS.

**Comment by Ezsk** [comment]
> I'm not concerned with what values other items use.

It’s hard to say but, It is not logical.
There is no other way to determine a value that doesn't exist in real like a encumbrance than by comparing it with the in-game item. I'm sure you had determined the value compared with something in the game.

> The SPCS is relatively heavy at 10kg but it has quite good weight distribution.

The actual body armor performance evaluation report is a relative evaluation with the actual other body armor. It is not a comparison with shirts and jackets.

**Comment by glenmack** [comment]
> I'm not concerned with what values other items use. We can and should derive encumbrance values from real information about the item. The SPCS is relatively heavy at 10kg but it has quite good weight distribution. It's pretty well designed for wearer comfort and the descriptions of its performance in SPD-VII indicate that the impact of 10 encumbrance closely reflects the impact of wearing the SPCS.

Perceived "realism" with entire disregard for game balance is a disturbing trend, and entirely counter-productive. All this stuff is meant to protect from bullets in a game where "realism" indicates that sub machine guns are entirely ineffective beyond 12m anyway.

You can't dump stuff into the game with disregard for the systems they are meant to work in because "realism". Though antiquated, the encumbrance system is largely there to ensure you can't fight like you're Neo while wearing 50 layers of gear. Part of making this any sort of worthwhile game is maintaining a sliding scale where certain armors are suitable for certain situations. That means considering where your values are in terms of similar items.

**Comment by Warriorcat49** [comment]
> Not to mention that Bash armor 50 also makes pretty much most bashing melee attacks completely irrelevant if they happen to hit the armor, which is not really how most ballistic armor tend to work. Nor should they be protecting from melee attacks from heavier bladed melee weapons either.

This is exactly how modern rifle-rated ballistic armor works. Rifle plates (level III and up) are made of either thick (6-12mm), hard, modern steel, or thick, very hard ceramic backed with inflexible, resin-impregnated Kevlar. Bashing attacks which strike the plates will knock the wind out of you at worst because the impact is spread out over the entire surface area of the plate, and even heavy bladed weapons won't do anything but scratch the surface. Remember that medieval jousting gear was capable of stopping lance hits from horseback (the heaviest hit from a hand-held bladed weapon I can think of), and it was essentially just 3-6mm carburized mild steel. Now, that in particular would put you on your back, but that has nothing to do with the plate being defeated.

**Comment by anothersimulacrum** [comment]
> sub machine guns are entirely ineffective beyond 12m anyway.

Tiles are not meters, you cannot use them to prescribe range to guns.

**Comment by ymber** [comment]
Unless someone presents evidence that this item doesn't accurately reflect the properties of an SPCS this issue isn't valid.

**Comment by Ezsk** [comment]
> Unless someone presents evidence that this item doesn't accurately reflect the properties of an SPCS this issue isn't valid.

I want to say the burden of proof is on the person who created the ballistic vest. I don't requesting reopen here though.

**Comment by davidpwbrown** [comment]
The bottom line is, if this item dosnt compare equally with the other items, and this item is thought out and correct values, then it is the other items that are wrong, and they need to be changed and they probably will.

Talking about "ruining game balance" and putting an item in that "disrupts all the other items" is nonsensical in the experimental version of the game.

If we waited until *all* the items were updated before changing them all in one big go, then it would never get done, this is what you get with incremental development, this is not a release of a periodic commercially available game that is guaranteed to be balanced at all release points.
